src/Stratis.Bitcoin.Features.RPC/Controllers/FullNodeController.cs

```diff
@@ -175,6 +177,44 @@ public BlockHeaderModel GetBlockHeader(string hash, bool isJsonFormat = true)
             return model;
         }
 
+        /// <summary>
+        /// Returns information about a bitcoin address
+        /// </summary>
+        /// <param name="address">bech32 or base58 BitcoinAddress to validate.</param>
+        /// <returns>JObject containing a boolean indicating address validity</returns>
+        [ActionName("validateaddress")]
+        [ActionDescription("Returns information about a bech32 or base58 bitcoin address")]
+        public JObject ValidateAddress(string address)
```

Comment:
@DanGould Can you please explain why you chose to return a JObject instead of a JSonResult with the boolean value in it? As far as I know JsonResults are far easier to interpret by a json ajax call etc. Please correct me if I'm wrong ;)

Reply:
I was writing this method particularly for this use case, which is why I used JObject. I can see JsonResult being friendlier for other applications. I haven't yet been able to get the noted method to work by returning JsonResult. It would make sense for that Log in (RPCArgs.cs)[https://github.com/BreezeHub/BreezeProject/blob/97c3c96bc238b616cabf7cd0a955e1372b871fc0/NTumbleBit/Configuration/RPCArgs.cs] to use a NBitcoin method for validateaddress anyway, so it probably should be migrated to JsonResult

```
try
			{
				var address = new Key().PubKey.GetAddress(network);
				var isValid = ((JObject)rpcClient.SendCommand("validateaddress", address.ToString()).Result)["isvalid"].Value<bool>();
				if(!isValid)
				{
					Logs.Configuration.LogError("The RPC Server is on a different blockchain than the one configured for tumbling");
					throw new ConfigException();
				}
			}
```

Reply:
@DanGould Ah I see OK! 

JsonResult is definitely what should be used here but yeah it won't work if Breeze expects a JObject (fail)

Can you perhaps maybe submit a PR to change that? And then once done fix this again?

In the meantime we can push this as is but just create an issue in BreezeHub 👍